src/commands/Info/levels.ts

```diff
@@ -111,7 +111,13 @@ function fetchPage(page: number, pages: DbUser[][], getString: GetStringFunction
 		author: { name: getString("moduleName") },
 		title: getString("pageTitle"),
 		footer: {
-			text: getString("pagination.page", { variables: { number: page + 1, total: pages.length }, file: "global" }),
+			text: getString("pagination.page", {
+				variables: {
+					number: formatNumberToLocaleString(page + 1, getString),
+					total: formatNumberToLocaleString(pages.length, getString),
```

Comment:
why is this needed?

Reply:
`formatNumberToLocaleString' formats to a locale string. If there are no changes with small numbers in some languages, it changes in Arabic.

en: 123
ar: ١٢٣

Reply:
That change seems out of the scope of this PR and honestly I’m not sure if we’d wanna go down that path.

Reply:
> That change seems out of the scope of this PR and honestly I’m not sure if we’d wanna go down that path

If you won't format it everywhere it will be formated to locale string in some strings and in others it won't.